### ISSUE TITLE
Twitter couter fails with getaddrinfo ENOTFOUND 

### ISSUE THREAD

**Issue by shershen08** [Globobeet/sociare-counter#10]
In my sources I've tried to include all available sharing services:

`var sources_use = ['facebook', 'twitter', 'pinterest', 'linkedin', 'googleplus'];`. 

Then process fails with following stacktrace:
`Error getting count: RequestError: Error: getaddrinfo ENOTFOUND urls.api.twitter.com urls.api.twitter.com:443`

Just to to reproduce, links I was passing are OK formatted and those pages are available: such as 
http://mashable.com/2016/04/06/ice-cube-ellen-rap-songs/, https://css-tricks.com/debouncing-throttling-explained-examples/, https://www.smashingmagazine.com/2016/04/consider-react-native-mobile-app/

Documentation or example does not mention that access to social networks must be somehow configured with API key or other details.

**Comment by joshlasdin** [comment]
@shershen08 Thanks for posting this. I believe this is due to [Twitter shutting down their share count API](https://blog.twitter.com/2015/hard-decisions-for-a-sustainable-platform) back in November 2015. I was hoping to find a suitable work around for getting Twitter counts but should probably put some info/deprecation notices in the code & documentation.

To my recollection, none of the other services required any sort of authentication or API credentials. 

Hope this helps, and apologies for not being more pro-active with the twitter issue :)

**Comment by shershen08** [comment]
Yes all other services works really smooth for me! But my point was to remove twitter from README example and ad a not maybe. I can do a PR if you will

**Comment by joshlasdin** [comment]
I'd be happy to review a PR, thanks!

**Comment by shershen08** [comment]
@joshlasdin  Did you have time to check that one https://github.com/shershen08/sociare-counter/commit/1fa25fd7eb1ef1007157a0a547fb16d346271280 ?

**Comment by joshlasdin** [comment]
@shershen08 Thanks, just left you a few (minor) comments. Other than that it looked good. Let me know once you've updated and I'll be sure to merge & publish

**Comment by shershen08** [comment]
I've updated the tests for build to pass, see - https://github.com/shershen08/sociare-counter/commits/issue10

but now 100% coverage fails:

```
=============================== Coverage summary ===============================
Statements   : 99.77% ( 425/426 ), 89 ignored
Branches     : 100% ( 208/208 ), 88 ignored
Functions    : 98.73% ( 78/79 )
Lines        : 98.11% ( 52/53 )
================================================================================
> sociare-counter@0.1.7 posttest /home/travis/build/shershen08/sociare-counter
> istanbul check-coverage --statement 100 --function 100 --branch 100 --line 100
/home/travis/build/shershen08/sociare-counter/node_modules/istanbul/lib/cli.js:38
        throw ex; // turn it into an uncaught exception
        ^
ERROR: Coverage for statements (99.77%) does not meet global threshold (100%)
ERROR: Coverage for lines (98.11%) does not meet global threshold (100%)
ERROR: Coverage for functions (98.73%) does not meet global threshold (100%)
npm ERR! Test failed.  See above for more details.
```

what to do with this?

**Comment by joshlasdin** [comment]
@shershen08 That makes sense. I'll try and take a look later today, thanks!

**Comment by joshlasdin** [comment]
@shershen08 Just published  [0.2.0](https://www.npmjs.com/package/sociare-counter) which removes the Twitter handlers & updates the documentation explaining why. Thanks again for your help with this!
